### DISCUSSION TITLE
Keep it simple

### DISCUSSION THREAD

**Discussion by chepalitos** [xournalpp/xournalpp#3262] [Ideas]
Keep it simple or leave a "litle" version for the ones that liked this app from the very begining.
Very nice improvemetns recently but I woudnt lost the idea of a simple, fuild and beautifull handwriting app.

**Comment by Technius** [comment]
Which features in particular seem to be too complex? The basic usage and function is still very simple in my opinion; any more complex features are entirely optional to use. And if Xournal++ is too complex, it is also possible to switch back to Xournal.

**Reply by chepalitos** [reply]
Hello!
Sorry, I was a little late.
It is a personal idea. Not that the functions are complicated. It is the idea that adding many features that are very particular to specific uses could become something that makes the daily use of the application a little more complicated. Just an idea and a simple suggestion.
Thanks.

**Reply by Technius** [reply]
We tend to reject feature requests with very specific use cases. However, users are free to customize the application to their needs by writing custom plugins.
